06-building-a-program-start-to-finish/setting-up-linters/output/README.md

```diff
@@ -0,0 +1,8 @@
+#### Output
```

Comment:
Okay @q-ode. I have done both. Improved the ReadMe and added the links to the images.